locale/en/docs/guides/event-loop-timers-and-nexttick.md

```diff
@@ -412,9 +412,7 @@ added, which means every day we wait, more potential breakages occur.
 While they are confusing, the names themselves won't change.
 
 *We recommend developers use `setImmediate()` in all cases because it's
-easier to reason about (and it leads to code that's compatible with a
-wider variety of environments, like browser JS.)*
-
+easier to reason about.*
```

Comment:
I think the blank line between the paragraph and the header might be required? Regardless, it makes the raw markdown easier to read. Let's keep it.

```suggestion
easier to reason about.*

```

Reply:
I've helped to add it, thanks for your tip ;)